lib/sass/script/functions.rb

```diff
@@ -196,11 +196,16 @@ module Sass::Script
   # Maps in Sass are immutable; all map functions return a new map rather than
   # updating the existing map in-place.
   #
-  # \{#map_get map-get($map, $key)}
-  # : Returns the value in a map associated with a given key.
+  # \{#map_get map-get($map, $keys...)}
+  # : Returns the value in a map (or nested map) associated with a given series of keys.
   #
   # \{#map_merge map-merge($map1, $map2)}
   # : Merges two maps together into a new map.
+  # \{#map_merge map-merge($map1, $keys..., $map2)}
+  # : Merges a map into another map, nested with the specified keys
```

Comment:
Here I would just not mention nesting. It's a narrow enough use-case that it can be described in the body documentation only.